asciidoc/volume1/tf1-ch-10-sdpi-p.adoc

```diff
@@ -581,6 +619,89 @@ Future versions *_MAY_* include options such as the following (not in priority o
 
 |===
 
+Options that may be selected for this Integration Profile are listed in the <<vol1_table_sdpi_p_actors_transactions>> along with the actors to which they apply.
+Dependencies between options when applicable are specified in
+notes.
+
+{empty} +
+
+[#vol1_table_sdpi_p_actors_transactions]
+.SDPi-P Profile - Actors and Options
+[%autowidth]
+[cols="1,1,1"]
+|===
+.^|Actor
+.^|Option Name
+.^|Vol. & Section
+
+| <<vol1_spec_sdpi_p_actor_somds_consumer>>
+| <<vol1_spec_sdpi_p_option_discovery_proxy>>
+| <<vol1_clause_sdpi_p_option_discovery_proxy>>
+
+| <<vol1_spec_sdpi_p_actor_somds_discovery_proxy>>
+| <<vol1_spec_sdpi_p_option_discovery_proxy>>
+| <<vol1_clause_sdpi_p_option_discovery_proxy>>
+
+| <<vol1_spec_sdpi_p_actor_somds_provider>>
+| <<vol1_spec_sdpi_p_option_discovery_proxy>>
+| <<vol1_clause_sdpi_p_option_discovery_proxy>>
+
+|===
+
+[#vol1_clause_sdpi_p_option_discovery_proxy]
+==== Discovery Proxy Option
+[#vol1_spec_sdpi_p_option_discovery_proxy, reftext='Discovery Proxy Option']
+
+#**<add high-level description of DP option, including sequence diagram, requirements, rationale, behaviour, etc.>
+**#
+
+Rationale:
+
+* "Ad Hoc" discovery is the default way in which SOMDS_Participant Actors manage their network presence and departure;
+* This default mode uses the DEV-23, DEV-24 and DEV-34 transactions, all of which rely on multicast services;
+* Some network configurations do not support or allow multicast services, requiring an alternative means of discovery and network presence and departure management;
+* The SOMDS Discovery Proxy Actor provides an alternative means for managing SOMDS_Provider Actor presence and departure from a network that does not require the use of multicast services;
+* Some configurations may allow network configurations that simultaneously support both "ad hoc" (default) and proxy-based discovery
+
+<<figure_sdpi-p-discovery-proxy-option-sequence-diagram>> provides an overview of the interactions of the SOMDS Consumer and Somes Provider Actors with the SOMDS Discovery Proxy Actor.
+
+.SDPi-P Discovery Proxy Option - Transaction Sequence Diagram
+[#figure_sdpi-p-discovery-proxy-option-sequence-diagram]
+[plantuml, target=target=puml-sdpi-p-discovery-proxy-option-sequence-diagram, format=svg, format=svg]
+....
+include::../plantuml/vol1-figure-sdpi-p-discovery-proxy-option-sequence-diagram.puml[]
+....
+
+##David - how much of the information below is in the current TF-2 & TF-2A detail?  TBD whether the requirements should be here at a more generic level or there.  Theory of Operation and non-transaction-specific requirements should go in this section;
+##
+
+* SOMDS_Provider systems shall support use of the SOMDS_Discovery_Proxy Actor (DEV-46)
```

Comment:
I don't think we need these particular requirement if we have a conditional "proxy option enabled" requirement.

Reply:
So the question is one of how to make sure that formal requirements for conformity assessment (i.e., exported to the JSON file) are captured based on the contents of the actors & transactions + Actors & options tables.  Note 2 to the Transactions table indicates that the transaction is required IF the actor is configured to use a DP.  Of course, they are always required for the DP actor.  Question is where to put the requirements metadata that will then get exported to JSON.  

Option would be to attach it to the note reference in the transaction table cell associated with the actor+transaction pair.  OR to some other text in that row.    OR I could put the formal requirement above with the associated text.  

For this pass, I'll just have a simple descriptive text about the obvious requirements and assume that the formal RI metadata will be contained in the actor+transactions table, since that's where we'll have to have the other similar requirements included